backend/benefit/applications/tests/test_applications_api.py

```diff
@@ -1153,6 +1153,28 @@ def test_application_accept(
     assert handling_application.calculation.granted_as_de_minimis_aid is True
 
 
+def test_application_with_batch_back_to_handling(
+    request,
+    handler_api_client,
+    decided_application,
+):
+    """
+    granted_as_de_minimis_aid is set at the same time the application is accepted.
```

Comment:
Is this comment up to date? There is nothing in this test which directly mentions granted_as_de_minimis_aid other than this comment.

Reply:
good catch, fixed